lib/routes/avatar/delete.js

```diff
@@ -4,12 +4,40 @@
 
 const hex = require('buf').to.hex;
 const Joi = require('joi');
+const P = require('../../promise');
 
 const AppError = require('../../error');
+const config = require('../../config');
 const db = require('../../db');
+const logger = require('../../logging')('routes.avatar.delete');
+const request = require('../../request');
 const validate = require('../../validate');
 
+const WORKER_URL = config.get('worker.url');
 const EMPTY = Object.create(null);
+const FXA_PROVIDER = 'fxa';
+
+function workerDelete(id) {
+  return new P(function(resolve, reject) {
+    var url = WORKER_URL + '/a/' + id;
```

Comment:
There's now two places where we have "build a worker URL and make a request to it logic"; we should consider a follow-up refactor to better encapsulate some of this shared logic.